### PR TITLE
fix(toolbar): remove glitch effect on hover in safari

### PR DESCRIPTION
**Pull request by AlexZakablukov**
**What this PR does** 📖
remove glitch effect on hover in safari

**Which issue(s) this PR fixes** 🔨
[AP-1210](https://satellite-im.atlassian.net/browse/AP-1210)

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

File: components/ui/ComingSoon/ComingSoon.less [hunk 1]
```diff
@@ -11,9 +12,11 @@
       justify-content: center;
     }
   }
-  .grayscaled {
-    -webkit-filter: grayscale(100%); /* Safari 6.0 - 9.0 */
-    filter: grayscale(100%);
+  &.grayscaled {
+    .circle{
+      -webkit-filter: grayscale(100%); /* Safari 6.0 - 9.0 */
```
Comment (stavares843) [inline components/ui/ComingSoon/ComingSoon.less:17]: I know this was already there but do we need this? The last safari version is 15.0

I don't think we need to add support for such old versions

Comment (molimauro) [reply components/ui/ComingSoon/ComingSoon.less:17]: Sara is right, we can remove it

Comment (AlexZakablukov) [reply components/ui/ComingSoon/ComingSoon.less:17]: ok, i agree with you)

Comment (AlexZakablukov) [reply components/ui/ComingSoon/ComingSoon.less]: fixed

**Review by stavares843 — APPROVED** [review]

**Pull request merged**